portfolio/src/main/webapp/index.html

```diff
@@ -25,84 +26,169 @@
+          <form id="new-event-form">
+            <div class="form-group">
+              <label for="new-event-name">Event name:</label>
+              <input
+                type="text"
+                class="form-control"
+                aria-describedby="new-event-name"
+                id="new-event-name"
+                name="new-event-name"
+                value="New Event"
+              />
+            </div>
 
-    <div id="schedule-result-wrapper">
-      <h2>Your tasks are scheduled!</h2>
+            <div class="row">
+              <div class="col">
+                <div class="form-group">
+                  <label for="new-event-start-time">Start time:</label>
+                  <input
+                    type="time"
+                    class="form-control"
+                    aria-describedby="new-event-start-time"
+                    id="new-event-start-time"
+                    name="new-event-end-time"
+                    value="09:00"
+                  />
+                  <small id="new-event-time-help-text" class="form-text text-muted"
+                    >The times you enter will be in your current timezone.</small
+                  >
+                </div>
+              </div>
+              <div class="col">
+                <div class="form-group">
+                  <label for="new-event-end-time">End time:</label>
+                  <input
+                    type="time"
+                    class="form-control"
+                    aria-describedby="new-event-end-time"
+                    id="new-event-end-time"
+                    name="new-event-end-time"
+                    value="17:00"
+                  />
+                </div>
+              </div>
+            </div>
+
+            <button type="button" class="btn btn-primary" onclick="createNewCalendarEventUserInput()">
+              Add this event
+            </button>
+
+            <div class="alert alert-secondary" role="alert" id="event-end-time-warning">
+              End time must be later than start time.
+            </div>
+            <div class="alert alert-secondary" role="alert" id="event-duplicate-warning">
+              This event has already been already added.
+            </div>
+          </form>
+
+          <div id="new-event-list-wrapper">
+            <h2>Events added:</h2>
+            <div id="new-event-list"></div>
+          </div>
+        </div>
+
+        <div id="calendar-view-wrapper" class="col-md-12 col-lg-6 col-xl-3">
+          <h4>Calendar view will be embedded here</h4>
+        </div>
+
+        <div id="schedule-result-wrapper" class="col-md-12 col-lg-6 col-xl-3">
+          <button type="button" class="btn btn-primary" onclick="startScheduling()">
+            Start scheduling!
+          </button>
+          <h2>Your tasks are scheduled here!</h2>
+          <div id="schedule-result-list"></div>
+        </div>
+      </div>
     </div>
```

Comment:
This div closing tag is unmatched so I think we can remove this or something is missing from the file earlier.

Reply:
Done.